docs/pages/database-access/guides/gui-clients.mdx

```diff
@@ -353,3 +353,75 @@ Click `Add Redis Database`.
 Congratulations! You have just connected to your Redis instance.
 
 ![Redis Insight Connected](../../../img/database-access/guides/redis/redisinsight-connected.png)
+
+
+## Snowflake - JetBrains (IntelliJ, Goland, DataGrip, PyCharm etc.)
+
+<Admonition type="note">
+  Snowflake integration works only in the authenticated proxy mode. Remember to add `--tunnel` flag when starting the proxy, example:
+  ```
+  tsh proxy db --tunnel --port 2000 snowflake
+  ```
+</Admonition>
+
+In "Database Explorer" click and the add button and then pick "Data Source" and pick "Snowflake":
+
+![JetBrains Add Database](../../../img/database-access/guides/snowflake/jetbrains-add-database.png)
+
+Next set "Host" to `localhost` and "Port" as the one returned by the proxy command (`2000` in the example above).
+Set the "Username" to match the one that you are connecting to using Teleport and enter any value in the "Password" field:
```

Comment:
```suggestion
Set the "Username" to match the one that you are assuming when you connect to Snowflake via Teleport and enter any value in the "Password" field:
```

Reply:
If the password field will be disregarded, I don't think it makes sense to advise the reader to enter any value. Is it okay to leave this blank?